docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -8,19 +8,22 @@ aliases:
   - ../../../enterprise/saml/enable-saml/
   - ../../../enterprise/saml/set-up-saml-with-okta/
   - ../../../enterprise/saml/troubleshoot-saml/
-description: Learn how to configure SAML authentication in Grafana.
-menuTitle: Configure SAML authentication
-title: Configure SAML authentication in Grafana
+description: Learn how to configure SAML authentication in Grafana's configuration file.
+menuTitle: Configure SAML using the configuration file
+title: Configure SAML authentication using the configuration file
 weight: 1100
 ---
 
-# Configure SAML authentication in Grafana
+# Configure SAML authentication using the configuration file
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
 
 SAML authentication integration allows your Grafana users to log in by using an external SAML 2.0 Identity Provider (IdP). To enable this, Grafana becomes a Service Provider (SP) in the authentication flow, interacting with the IdP to exchange user information.
 
-The SAML single sign-on (SSO) standard is varied and flexible. Our implementation contains a subset of features needed to provide a smooth authentication experience into Grafana.
+You can configure SAML authentication in Grafana through the user interface (UI) or the Grafana configuration file. For instructions on how to set up SAML through Grafana's UI, refer to [Configure SAML authentication using the Grafana user interface]({{< relref "../saml-ui/" >}}).
+Both methods offer the same configuration options, but you might prefer using the Grafana configuration file if you want to keep all of Grafana's authentication settings in one place. Hosted Grafana users do not have access to Grafana configuration file, so they should configure SAML through Grafana's UI.
```

Comment:
Similar to the comment above, perhaps using Grafana Cloud instead of Hosted Grafana makes more sense.